src/Shared/CertificateGeneration/UnixCertificateManager.cs

```diff
@@ -18,7 +18,12 @@ internal UnixCertificateManager(string subject, int version)
     {
     }
 
-    public override bool IsTrusted(X509Certificate2 certificate) => false;
+    public override bool IsTrusted(X509Certificate2 certificate)
+    {
+        using X509Chain chain = new X509Chain();
+        chain.ChainPolicy.RevocationMode = X509RevocationMode.NoCheck;
+        return chain.Build(certificate);
```

Comment:
Is there any reason not to use the logic on all platforms? It looks like on Windows we check for the cert in `new X509Store(StoreName.Root, StoreLocation.CurrentUser)` specifically and on macOS we use `security verify-cert -p basic -p ssl` on the exported dev cert. I'm not sure what advantages either of those things have over this. @javiercn

Reply:
This checks the result rather than the implementation. It can be used on all platforms.

Reply:
I'll follow up with @javiercn separately and update the other implementations if appropriate.

Reply:
Can we move this down to CertificateManager.cs then and make it non-virtual/abstract?